inference_perf/config.py

```diff
@@ -26,10 +26,22 @@ class APIType(Enum):
 class DataGenType(Enum):
     Mock = "mock"
     ShareGPT = "shareGPT"
+    Synthetic = "synthetic"
+
+# Represents the distribution for input prompts and output generations.
+class Distribution(BaseModel):
+    min: int = 10
+    max: int = 1024
+    mean: float = 512
+    std_dev: float = 200
+    total_count: int = 1000
 
 
 class DataConfig(BaseModel):
     type: DataGenType = DataGenType.Mock
+    # Distributions are only supported for synthetic dataset at this moment
+    input_distribution: Optional[Distribution] = Distribution
+    output_distribution: Optional[Distribution] = Distribution
```

Comment:
Whats the purpose of having `output_distribution` here?

Reply:
It can be used to limit max tokens on certain APIs?

Reply:
Yes, this is to add output distribution next. We can control the output tokens generated by setting max_tokens and using ignore_eos to force it as needed.